### PR TITLE
feat: add titles and defaults to network pages

### PR DESCRIPTION
**Pull request by olivierwilkinson**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready | [OS-1214](https://pi-top.atlassian.net/browse/OS-1214) |

#### Main changes
Add a centred title and underline to the network pages.

Remove the username and password from the USB network page. Login
information will be accessible through the System menu soon.

Add smaller variants of networking icons to make the pages feel less
cramped. Keep the larger variants for use in other areas.

Replace references to "Access Point" with "Wi-Fi Hotspot" since that is
a more descriptive name.

Ensure error and disabled states are friendly:
- when Wi-Fi is not connected use "Not connected" instead of "error"
- when AP mode is disabled say "Not active" instead of staying blank

### CODE REVIEW

File: pt_miniscreen/pages/network/ap.py [hunk 1]
```diff
@@ -9,16 +9,16 @@ class Page(PageBase):
...
                 text=lambda: get_ap_mode_status().get("passphrase", ""),
             ),
             third_row=RowDataText(
-                icon_path="sys_info/networking/home.png",
+                icon_path="sys_info/networking/home-small.png",
                 text=lambda: get_ap_mode_status().get("ip_address", ""),
             ),
         )
-        super().__init__(size=size, row_data=row_data)
+        super().__init__(size=size, row_data=row_data, title="Access Point")
```
Comment (angusjfw) [inline pt_miniscreen/pages/network/ap.py:24]: I think 'Wi-Fi Hotspot' or maybe even just 'Hotspot' if that doesn't fit is better. people are more likely to get it imo and I think that's what we call it in onboarding now

File: pt_miniscreen/pages/network/ap.py [hunk 2]
```diff
@@ -9,16 +9,16 @@ class Page(PageBase):
     def __init__(self, size):
         row_data = NetworkPageData(
             first_row=RowDataText(
-                icon_path="sys_info/networking/antenna.png",
-                text=lambda: get_ap_mode_status().get("ssid", ""),
+                icon_path="sys_info/networking/antenna-small.png",
```
Comment (angusjfw) [inline pt_miniscreen/pages/network/ap.py:12]: i think this should be the wifi icon, same as we use for ssid on the wifi page

**Review by angusjfw — COMMENTED** [review]
Looks so awesome!

File: pt_miniscreen/pages/network/ap.py (hunk 1 above)
Comment (angusjfw) [reply pt_miniscreen/pages/network/ap.py:24]: actually it does say 'Access Point' on the settings page, so if we change one it should be both

Comment (olivierwilkinson) [reply pt_miniscreen/pages/network/ap.py]: Yeah I agree 'Wi-Fi Hotspot' is a better name. I've updated it in both places :)

File: pt_miniscreen/pages/network/ap.py (hunk 2 above)
Comment (olivierwilkinson) [reply pt_miniscreen/pages/network/ap.py]: Yeah agreed! Done :)

File: pt_miniscreen/pages/network/wifi.py [hunk 3]
```diff
@@ -12,19 +12,19 @@ class Page(PageBase):
...
         row_data = NetworkPageData(
             first_row=RowDataGeneric(
-                icon_path="sys_info/networking/antenna.png",
+                icon_path="sys_info/networking/antenna-small.png",
                 hotspot_type=WifiStrengthHotspot,
             ),
             second_row=RowDataText(
-                icon_path="sys_info/networking/wifi.png",
+                icon_path="sys_info/networking/wifi-small.png",
                 text=get_wifi_network_ssid,
```
[the remark this answers is not in the document]
Comment (olivierwilkinson) [reply pt_miniscreen/pages/network/wifi.py]: I went with "Not connected" :)

File: pt_miniscreen/pages/network/wifi.py [hunk 4]
```diff
@@ -10,21 +10,28 @@
 
 class Page(PageBase):
     def __init__(self, size):
+        def get_ssid():
+            ssid = get_wifi_network_ssid()
+            if ssid == "error":
```
Comment (jcapona) [inline pt_miniscreen/pages/network/wifi.py:15]:
```suggestion
            if ssid == "Error":
```

Comment (olivierwilkinson) [reply pt_miniscreen/pages/network/wifi.py:15]: good catch!

**Review by jcapona — APPROVED** [review]
network pages look awesome, great job!

**Pull request merged**